app/Models/WaitingRoom.php

```diff
@@ -30,6 +30,24 @@ public function users()
     }
 
     /**
+     * Gets a CSV export of all the users in this waiting room.
+     *
+     * @return \League\Csv\ $csv
+     */
+    public function getCSVExport()
```

Comment:
hmmm could the helper just take in an array of the data that the you want to export? That way it can be flexible, and the controller would just be responsible for creating that array of data.